### DISCUSSION TITLE
Add interaction with the Windows system tray.

### DISCUSSION THREAD

**Discussion by Ariel-MN** [Embarcadero/DelphiVCL4Python#27] [Ideas]
While in Delphi we have **TTrayIcon** that combined with **TPopupMenu** does an excellent job adding applications to the system tray, TTrayIcon even allows to send Windows notifications. I would like to suggest adding these two classes.

**Comment by lmbelo** [comment]
Hi @Ariel-MN,
I'm including your suggestion to our roadmap.
